stories/SummaryDetail.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { SummaryDetail } from '../src';
+import './utils/global.css';
+
+const stories = storiesOf('Summary Detail', module);
+
+stories
+  .add('Base', () => (
+    <div>
+      <SummaryDetail
+        renderOpenContent={() => <div>Summary Content</div>}
```

Comment:
From an API perspective, it would probably make more sense to pass the components' content as a node, maybe even as `children`. A function prop feels a bit clunky here. I would expect `title` to be it's own prop since I do not want to know about the implementation details (h3 and classes). `children` IMO is the wrong place for the title. I have two alternative suggestions:

``` jsx
// renderTitle is optional
<SummaryDetail title="foo" renderTitle={title => title}>
  {({ isOpen }) => 'foo')}
</SummaryDetail>
```

``` jsx
// Children is only rendered when `isOpen`
<SummaryDetail title="foo" renderTitle={title => title}>
  foo
</SummaryDetail>
```

Reply:
You are right. Suggestion 1) is nice, it will prevent closed content from being rendered